src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,111 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD = 6;
+    private static final int STORAGE_LIST_SIZE = 2;
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setId(321123L);
+        user.setPassword("hawkIsGood");
+        user.setLogin("SomeLogin");
+        user.setAge(30);
+    }
+
+    @Test
+    void register_Young_IsNotOk() {
+        user.setAge(16);
+        Assertions.assertThrows(RuntimeException.class, () -> registrationService.register(user),
+                "Expected exception when user age is less than required");
+    }
+
+    @Test
+    void register_Old_IsOk() {
+        User register = registrationService.register(user);
+        assertTrue(register.getAge() > MIN_AGE, "Good! You age perfect for registration");
+    }
+
+    @Test
+    void registerAge_Null_IsNotOk() {

```

Comment:
```suggestion
    void register_ageNull_isNotOk() {

```
Correct all names along these lines